front/app/containers/Admin/projects/project/analysis/Insights/Files/FileSelectionView/index.tsx

```diff
@@ -107,6 +108,17 @@ const FileSelectionView = ({ setIsFileSelectionOpen, analysisId }: Props) => {
             options={fileOptions}
             placeholder={formatMessage(messages.attachFilesFromProject)}
           />
+
+          <Box mt="24px">
+            <ButtonWithLink
+              linkTo={`/admin/projects/${projectId}/files`}
+              buttonStyle="text"
+              icon="upload-file"
+              openLinkInNewTab={true}
+            >
+              {formatMessage(messages.uploadFiles)}
+            </ButtonWithLink>
```

Comment:
Just adds a helpful link to the Files Tab within the sense-making "Add a file as context" UI.